packages/core/template/default/index.md

```diff
@@ -9,124 +9,45 @@
 
 <div class="bg-primary text-white py-5 mb-4">
   <div class="container">
-    <h1 class="display-4 no-index">Landing Page Title</h1>
-    <p class="lead">A tagline can go here</p>
+    <h1 class="display-4 no-index">Great!<br>You just initialized a MarkBind site.</h1>
+    <p class="lead">Let's get started...</p>
   </div>
 </div>
 
-# Heading 1
-Some text some text some text some text some text some text some text. **Some text some text some text some text some text ==some text== some text**. Some text some text some text some text some text some text some text some text some text some text some text some text some text some text. Some text some text some text some text some text some text. Some text some text some text some text some text some text some text.
+<hr>
 
-**A block quote:**
+<h5><strong>User Guide: Authoring Contents</strong></h5>
```

Comment:
Instead of using pure html in the whole file, can we use a mix of html and markdown instead? This might be easier for first time users and is also one of our selling points. 

I think the headings should at least be done in markdown instead of html

Reply:
That's a good idea! I've changed the headings to markdown in [d5317d0](https://github.com/MarkBind/markbind/pull/2225/commits/d5317d095a7a908d8a582e36e54a7751e4889a53).